assets/script.js

```diff
@@ -63,7 +63,8 @@ $(document).ready(function() {
 //search submit
 function submitSearch(e) {
 	e.preventDefault();
-  	document.location = '/' + $('#lookup-box').val().toLowerCase();
+	var newLocation = $('#lookup-box').val().toLowerCase();
+  	document.location = encodeURIComponent((newLocation.length && newLocation[0] == '/') ? newLocation.slice(1) : newLocation);
```

Comment:
Haven't tested this but shouldn't it be `newLocation.slice(1) : '/' + newLocation` that way if a user does not enter a leading slash it will prepend it for them, eg if you type in `hash` and hit enter it would take you to `/hash`

Reply:
The slash appears to implied in the redirect - if we add one in that way we end up with two.
I've tested it as is and it's working for me, but have a play with it yourself if you like... it's your call! ;)

I seem to have messed up this PR now anyway with my merging antics, so I'll have to close it off and make a new one I think. 
Let me know how you want to proceed.

Reply:
@pfreitag Why should a user enter a leading slash? That should not be the default behavior. So we can either omit or escape it.

Reply:
@GaryStanton - Your code works for a URI on the root of the domain, but I think this would cause an issue if you were on a URL in a subfolder right? because without the leading slash in document.location it would assume you are navigating to another file within the same folder right? I haven't tested it but that is what I was assuming.

I was thinking something like this:

```
var newLocation = $('#lookup-box').val().toLowerCase();
document.location = encodeURIComponent((newLocation.length && newLocation[0] == '/') ? newLocation.slice(1) : '/' + newLocation);
```

If you want to close this PR and submit another that would probably be easier.

@shaedrich - this PR was to fix the case when someone types a slash into the text box, it was working fine otherwise.

Reply:
Well, I know. That's what I mentioned earlier: It works for that specific case, but we might think about how we should do it correctly because there might be other special cases which can cause bugs...

Reply:
@pfreitag - That code doesn't work for me. 
If I type `/cfimage` into the search box, it'll work fine, but if I type `cfimage` the browser redirects to `/%2Fcfimage`. My version works in both cases.
I can't find an example of a URL that's in a sub folder - which is to say every URL I can find in CFDocs looks like a folder off of the root.
This was really just a quick fix for a single case, but I agree with @shaedrich that we should look at a better way of handling the search as there coule be some other issues with various characters and combinations of characters. I opened up a related issue at #873.

Reply:
Ah, sorry - that was because the `/` is getting encoded with `encodeURIComponent` and turned into '%2F` ... This should work better:

```
var newLocation = $('#lookup-box').val().toLowerCase();
'/' + encodeURIComponent((newLocation.length && newLocation[0] == '/') ? newLocation.slice(1) : newLocation);
```

I agree that the search could be more robust, and there are probably some other character cases that are causing problems... Really it was just intended to be a jump to tag / function feature rather than a full text search, but a more robust search could certainly be useful.

Reply:
Hmm... could have sworn I replied to this last week! 
Yes, updated version works for me.